source/allmodules/allmodules.coffee

```diff
@@ -1,22 +1,23 @@
-import audioanalysermodule from "./audioanalysermodule"
-import audiomodule from "./audiomodule"
-import authoritysymbolmodule from "./authoritysymbolmodule"
-import backgroundmodule from "./backgroundmodule"
-import chatmodule from "./chatmodule"
-import configmodule from "./configmodule"
-import dancingvirusmodule from "./dancingvirusmodule"
-import debugmodule from "./debugmodule"
-import eventexchangemodule from "./eventexchangemodule"
-import messageboxmodule from "./messageboxmodule"
-import micbuttonmodule from "./micbuttonmodule"
-import peertopeermodule from "./peertopeermodule"
-import pianomodule from "./pianomodule"
-import qrdisplaymodule from "./qrdisplaymodule"
-import qrreadermodule from "./qrreadermodule"
-import startupmodule from "./startupmodule"
-import statemodule from "./statemodule"
-import systemsoundbuttonmodule from "./systemsoundbuttonmodule"
-import visualizationmodule from "./visualizationmodule"
+import * as audioanalysermodule from "./audioanalysermodule"
```

Comment:
this needs to be changed in toolset - this fully works with both ESM and CJS mods, so it's the best path forward imo